src/test/java/io/jenkins/pluginhealth/scoring/probes/LastCommitDateProbeIT.java

```diff
@@ -0,0 +1,28 @@
+package io.jenkins.pluginhealth.scoring.probes;
+
+import java.io.IOException;
+import java.time.ZonedDateTime;
+
+import io.jenkins.pluginhealth.scoring.AbstractDBContainerTest;
+import io.jenkins.pluginhealth.scoring.model.Plugin;
+import io.jenkins.pluginhealth.scoring.model.ProbeResult;
+import io.jenkins.pluginhealth.scoring.model.ResultStatus;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+public class LastCommitDateProbeIT extends AbstractDBContainerTest {
+
+    @Test
+    public void shouldReturnSuccessStatusOnValidSCM() throws IOException {
+        final Plugin plugin = new Plugin("parameterized-trigger", "https://github.com/jenkinsci/parameterized-trigger-plugin.git", ZonedDateTime.now())
+            .addDetails(ProbeResult.success("scm", "The plugin SCM link is valid"));
+        final LastCommitDateProbe probe = new LastCommitDateProbe();
+
+        final ProbeResult r = probe.apply(plugin);
+
+        assertThat(r.id()).isEqualTo("last-commit-date");
+        assertThat(r.status()).isEqualTo(ResultStatus.SUCCESS);
+    }
+}
```

Comment:
@alecharp I'm very sorry for delaying this PR for the Integration test you asked for 2 days ago. I see that you had to mark that comment as resolved because I didn't commit anything to it since then.

An integration would run everything from start to end, with no mocks at all. So, for that I have added `extends AbstractDBContainerTest` to this class so that we have a running container while we run this test. The aim of this integration test was to validate whether the cloned repo is getting deleted in the end or not.

So, I just simply copy pasted the unit test here in this class with dummy values. But my issue is how do I access the path of this plugin's repo inside this class? I'm asking this because, when I have access to the path of the repo, I would be able to use the method like `Files.exists()` to assert the absence of any data at the cloned-repo's path, and that would solve what we want to validate.

Reply:
sorry I was silent on this subject. 
let's not bring an integration test for the moment. The unit ones are already checking this. We will see down the road of the project if we have problems with this portion of code.

Reply:
Sounds good, thank you